### ISSUE TITLE
to_update = update_guild_commands[guild_id] KeyError:

### ISSUE THREAD

**Issue by colouredflower** [Disc0rdian/discord-boost-bot#2]
python3 main.py

 /$$$$$$$   /$$$$$$   /$$$$$$   /$$$$$$  /$$$$$$$$       /$$$$$$$   /$$$$$$  /$$$$$$$$
| $$__  $$ /$$__  $$ /$$__  $$ /$$__  $$|__  $$__/      | $$__  $$ /$$__  $$|__  $$__/
| $$  \ $$| $$  \ $$| $$  \ $$| $$  \__/   | $$         | $$  \ $$| $$  \ $$   | $$   
| $$$$$$$ | $$  | $$| $$  | $$|  $$$$$$    | $$         | $$$$$$$ | $$  | $$   | $$   
| $$__  $$| $$  | $$| $$  | $$ \____  $$   | $$         | $$__  $$| $$  | $$   | $$   
| $$  \ $$| $$  | $$| $$  | $$ /$$  \ $$   | $$         | $$  \ $$| $$  | $$   | $$   
| $$$$$$$/|  $$$$$$/|  $$$$$$/|  $$$$$$/   | $$         | $$$$$$$/|  $$$$$$/   | $$   
|_______/  \______/  \______/  \______/    |__/         |_______/  \______/    |__/   
                        mukitan | github.com/Mrpo16

Ignoring exception in on_connect
Traceback (most recent call last):
  File "/home/runner/discord-boost-bot/venv/lib/python3.8/site-packages/discord/client.py", line 352, in _run_event
    await coro(*args, **kwargs)
  File "/home/runner/discord-boost-bot/venv/lib/python3.8/site-packages/discord/bot.py", line 793, in on_connect
    await self.register_commands()
  File "/home/runner/discord-boost-bot/venv/lib/python3.8/site-packages/discord/bot.py", line 338, in register_commands
    to_update = update_guild_commands[guild_id]
KeyError: '943825721669865552'

![image](https://user-images.githubusercontent.com/95069650/193571566-440c477f-a153-49d1-b7c4-a1aac075f786.png)

**Comment by Disc0rdian** [comment]
> python3 main.py
> 
> Ignoring exception in on_connect Traceback (most recent call last): File "/home/runner/discord-boost-bot/venv/lib/python3.8/site-packages/discord/client.py", line 352, in _run_event await coro(*args, **kwargs) File "/home/runner/discord-boost-bot/venv/lib/python3.8/site-packages/discord/bot.py", line 793, in on_connect await self.register_commands() File "/home/runner/discord-boost-bot/venv/lib/python3.8/site-packages/discord/bot.py", line 338, in register_commands to_update = update_guild_commands[guild_id] KeyError: '943825721669865552'
> 
> ![image](https://user-images.githubusercontent.com/95069650/193571566-440c477f-a153-49d1-b7c4-a1aac075f786.png)

Are you sure you put the correct guild id in the settings

**Comment by colouredflower** [comment]
> > python3 main.py
> > Ignoring exception in on_connect Traceback (most recent call last): File "/home/runner/discord-boost-bot/venv/lib/python3.8/site-packages/discord/client.py", line 352, in _run_event await coro(*args, **kwargs) File "/home/runner/discord-boost-bot/venv/lib/python3.8/site-packages/discord/bot.py", line 793, in on_connect await self.register_commands() File "/home/runner/discord-boost-bot/venv/lib/python3.8/site-packages/discord/bot.py", line 338, in register_commands to_update = update_guild_commands[guild_id] KeyError: '943825721669865552'
> > ![image](https://user-images.githubusercontent.com/95069650/193571566-440c477f-a153-49d1-b7c4-a1aac075f786.png)
> 
> Are you sure you put the correct guild id in the settings

Yes, I have checked and copy pasted it multipule times but still no Changes
